contrib/k8s/examples/mysql/advanced/mysql-database-instance.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: example-mysql-database-instance
   namespace: default
 spec:
-  clusterServiceClassExternalName: azure-mysql-database
+  clusterServiceClassExternalName: azure-mysql-5-7-database
```

Comment:
Missed this (and other similar updates to the examples) in #407. Thanks for catching this and fixing it.